docs/dev/guide/custom-aa-tutorial.md

```diff
@@ -431,46 +429,45 @@ _Note, that zkSync has different address derivation rules from Ethereum_. You sh
 Before the deployed account can do any transactions, we need to top it up:
 
 ```ts
-await (await wallet.sendTransaction({
+await(
+  await wallet.sendTransaction({
     to: multisigAddress,
-    value: ethers.utils.parseEther('0.0001')
-})).wait();
+    value: ethers.utils.parseEther("0.0001"),
+  })
+).wait();
 ```
 
 Now, as an example, let's try to deploy a new multisig, but the initiator of the transaction will be our deployed account from the previous part:
 
 ```ts
-let aaTx = await aaFactory.populateTransaction.deployAccount(
-    salt,
-    Wallet.createRandom().address,
-    Wallet.createRandom().address
-);
+let aaTx = await aaFactory.populateTransaction.deployAccount(salt, Wallet.createRandom().address, Wallet.createRandom().address);
 ```
 
 Then, we need to fill all the transaction's fields:
 
 ```ts
 const gasLimit = await provider.estimateGas(aaTx);
-const gasPrice =  await provider.getGasPrice();
+const gasPrice = await provider.getGasPrice();
 
 aaTx = {
-    ...aaTx,
-    gasLimit: gasLimit,
-    gasPrice: gasPrice,
-    chainId: (await provider.getNetwork()).chainId,
-    nonce: await provider.getTransactionCount(multisigAddress),
-    type: 113,
-    customData: {
-        ergsPerPubdata: '1',
-        feeToken: utils.ETH_ADDRESS
-    } as Eip712Meta,
-    value: ethers.BigNumber.from(0)
-}
+  ...aaTx,
+  gasLimit: gasLimit,
+  gasPrice: gasPrice,
+  chainId: (await provider.getNetwork()).chainId,
+  nonce: await provider.getTransactionCount(multisigAddress),
+  type: 113,
+  customData: {
+    ergsPerPubdata: "1",
+    feeToken: utils.ETH_ADDRESS,
+  } as Eip712Meta,
+  value: ethers.BigNumber.from(0),
+};
 ```
 
 ::: tip Note on gasLimit
 
-Currently, we expect the `gasLimit` to cover both the verification and the execution step. Currently, the number of ergs that is returned by the `estimateGas` is `execution_ergs + 20000`, where `20000` is roughly equals to the overhead needed for the defaultAA to have both fee charged and the signature verified. In case your AA has very expensive verification step, you should add some constant to the `gasLimit`.    
+Currently, we expect the `gasLimit` to cover both the verification and the execution steps. Currently, the number of ergs that is returned by the `estimateGas` is
+`execution_ergs + 20000`, where `20000` is roughly equals to the overhead needed for the defaultAA to have both fee charged and the signature verified. In case your AA has very expensive verification step, you should add some constant to the `gasLimit`.
```

Comment:
I don't think so. Docs are rendered fine in preview. This change was made by `yarn fix:fmt` and it is much more easier and convenient to redact the docs since you don't need to scroll a lot of time to edit some word for example.